client/routes/Designer/components/index.js

```diff
@@ -47,29 +50,79 @@ const SurveyDesigner = (props) => {
   const changeActiveGroup = (grpIndex) => {
     updateActiveGroup(grpIndex);
   };
+  const addGroup = () => {
+    const newSurveyData = cloneDeep(surveyData);
+    newSurveyData.groups.push(newGroup);
+    updateSurveyData(newSurveyData);
+  };
+  const removeGroup = (e, index) => {
+    if (index === surveyData.groups.length - 1) {
```

Comment:
why this we can directly write ` e.stopPropagation();` no need of condition

Reply:
Let's  make `activeGroup` always zero